tools/loader/Loader.cpp

```diff
@@ -123,6 +124,19 @@ llvm::cl::opt<bool> dumpGraphOpt("dumpGraph",
                                  llvm::cl::desc("Prints Graph to stdout"),
                                  llvm::cl::cat(modelExportCat));
 
+llvm::cl::list<std::string> doNotConvertNodesOpt(
+    "do-not-convert-nodes",
+    llvm::cl::desc(
+        "Use to specify the name of nodes (e.g. Add, Div, etc.) that should "
+        "not be converted."),
+    llvm::cl::value_desc("NodeNames (e.g. Add,Div)"), llvm::cl::ZeroOrMore,
+    llvm::cl::CommaSeparated, llvm::cl::cat(loaderCat));
+
```

Comment:
The option "--do-not-convert-nodes" : Does it mean in most of cases, only a few of types of nodes should not be converted?  Just wondering why we don't use "do-convert-nodes" :)

Reply:
I don't have a good answer to this. I reproduced what we do for quantization and I assumed it was the same kind of use case.
What do you think?

Reply:
Yeah, the reason I went with `do-not` instead of `do` for quantization is that I was expecting we would quantize as much as possible, and so it's likely the `do-not` list is much shorter than the `do` list. E.g. for en2gr I was quantizing everything but Add, so it made more sense to do like this.